### PR TITLE
Homework: Documentation for tickets system

### PR DESCRIPTION
**Pull request by e-kondra**
Homework: Documentation for tickets system
Create a [BusTicketService] class with the following list of capabilities:
1) Create a [BusTicket] for a specified date and type.
2) Store a ticket in a in-memory storage.
3) Remove a ticket from the storage by ID.
4) Get ticket by ID
5) Search tickets by type and price (from price A to price B)
Cover the class and its function with Java docs where needed.
If you think that the code is “self-explainable” - don’t add docs.

### CODE REVIEW

**Comment by e-kondra** [comment]
As I understand, all methods in a new class are self-explainable and do not need documentation except method getTicketsByTypeAndPrice(), that may need some explanation

File: src/main/java/andersen/AndersenTasks/AndersenTasksApplication.java [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
-		Client client = new Client(ticket);
-		client.printRole();
-		client.getTicket();
 
-		Ticket ticket1 = new Ticket();
-		ticket1.setProtoId(12L);
-		System.out.println(ticket.equals(ticket1));
-		System.out.println(ticket.hashCode());
-		System.out.println(ticket1.hashCode());
+//		TicketService ticketService = new TicketService();
```
Comment (deolexx) [inline src/main/java/andersen/AndersenTasks/AndersenTasksApplication.java:50]: You should not keep this commented lines in code. The general purpose of VCS is that if you need some code from previous versions of file you could easily go and check  history in git

File: src/main/java/andersen/AndersenTasks/busticket/BusTicket.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class BusTicket{
+
+    private Long id;
+    private LocalDate startDate;
+    private BusTicketType ticketType;
+    private int price;
```
Comment (deolexx) [inline src/main/java/andersen/AndersenTasks/busticket/BusTicket.java:19]: not the best idea to use int for price field BigDecimal would be more suitable

**Review by deolexx — APPROVED** [review]
LGTM

**Pull request merged**